site/public/js/notebook_builder/widgets/markdown-widget.js

```diff
@@ -15,32 +15,33 @@ class MarkdownWidget extends Widget {
         const container = this.getContainer('Markdown');
         container.classList.add('markdown-widget');
 
-        // Add instructional link to the widget title area
-        const info_link = document.createElement('a');
-        info_link.setAttribute('href', 'https://submitty.org/student/communication/markdown');
-        info_link.setAttribute('title', 'Help using markdown');
-        info_link.setAttribute('target', '_blank');
-        info_link.innerHTML = '<i class="far fa-question-circle"></i>';
-        container.querySelector('.heading-container').appendChild(info_link);
+        const markdownArea = $(Twig.twig({
+            ref: "MarkdownArea"
+        }).render ({
+            markdown_area_id : `notebook-builder-markdown-${NUM_MARKDOWN}`,
+            markdown_area_query : `$('#notebook-builder-markdown-${NUM_MARKDOWN}')`,
+            markdown_area_name : '',
+            markdown_area_value : this.state.markdown_string,
+            placeholder : 'Enter text or markdown...',
+            preview_div_id : `notebook-builder-markdown-preview-${NUM_MARKDOWN}`,
+            preview_div_name : `notebook-builder-markdown-preview-${NUM_MARKDOWN}`,
+            preview_button_id : `notebook-builder-preview-button-${NUM_MARKDOWN}`,
+            onclick : `previewNotebookBuilderMarkdown.call(this, ${NUM_MARKDOWN})`,
+            render_buttons : true,
+            min_height : "100px",
+        }));
```

Comment:
We are ideally moving away from twig.js as it's not exactly super well supported, and has some number of idioscyrancies versus the mainline Twig.

It's very fine to just issue an AJAX call to fetch a rendered markdown area (presumably from the aforementioned `MarkdownController`).